LAS/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+using WellLog.Lib;
+using WellLog.Lib.DataAccess;
+using WellLog.Lib.Validators;
+
+namespace LAS
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //setup our DI
+            var serviceCollection = new ServiceCollection();
+            WellLogModule.RegisterTypes(serviceCollection);
+
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+
+            var lasLogFileDataAccess = serviceProvider.GetService<ILasLogFileDataAccess>();
+            var lasLog = lasLogFileDataAccess.Read("C:\\temp\\49009223560000.las");
+            foreach (var section in lasLog.Sections)
+            {
+                Console.WriteLine($"{section.SectionType}");
+
+                if (section.MnemonicsLines != null)
+                {
+                    foreach (var mnemonicLine in section.MnemonicsLines)
+                    {
+                        Console.WriteLine($"\tMnemonic: {mnemonicLine.Mnemonic}; Units: {mnemonicLine.Units}; Data: {mnemonicLine.Data}; Description: {mnemonicLine.Description}");
```

Comment:
You should move this code to another class where you can test how your data is being printed (since this is the purpose of this program).

Reply:
done in new pull request